packages/qwik/src/optimizer/src/qwik-binding-map.ts

```diff
@@ -30,15 +30,5 @@ export const QWIK_BINDING_MAP = {
         "platformArchABI": "qwik.win32-x64-msvc.node"
       }
     ]
-  },
-  "linux": {
-    "x64": [
-      {
-        "platform": "linux",
-        "arch": "x64",
-        "abi": "gnu",
-        "platformArchABI": "qwik.linux-x64-gnu.node"
-      }
-    ]
```

Comment:
This was removed automatically when building, should this change be reverted?

Reply:
Yes, please :smile: